content/community/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Community"
+weight: 100
+---
+
+# Contributing to Paketo Buildpacks
+
+## How can I contribute to the project?
+
+Welcome! We're glad you have an interest in improving Paketo Buildpacks.
+Firstly, check out our [Contribution
+Guide](https://github.com/paketo-buildpacks/.github/blob/main/CONTRIBUTING.md)
+for an idea of where to begin. We also have a collection of [good first
+issues](https://github.com/search?q=org%3Apaketo-buildpacks+org%3Apaketo-community+label%3A%22good+first+issue%22+state%3Aopen&type=Issues)
+on our repositories if you're eager to start contributing.
+
+### RFCs
+
+If you'd like to learn more about the buildpacks architecture and the design
+decisions being made, it may be helpful to read through some of our
+[project-wide RFCs](https://github.com/paketo-buildpacks/rfcs). Some language
+family buildpack repositories (e.g.
+[Node.js](https://github.com/paketo-buildpacks/nodejs/tree/main/rfcs)) also
+contain language-specific RFCs that may provide helpful context.
+
+### Discussions
+
+Do you have ideas for the future of the project? Well, we'd certainly like to
+hear them! Check out our [Discussions
+portal](https://github.com/paketo-buildpacks/feedback/discussions) to weigh in
+on ongoing conversations or to start a new one.
+
+
+## Connect with the community
+There are a few good ways to keep up with developments within the Paketo
+Buildpacks community:
+- Join our [working group
+  meetings](https://github.com/paketo-buildpacks/community#working-group-meetings)
+- Help improve docs: If you see something that's unclear or outdated, file an
```

Comment:
I'd suggest making this a little less verbose. Suggestion:
"Help improve our [docs](https://github.com/paketo-buildpacks/paketo-website/issues):"